docs/resources/lke_cluster.md

```diff
@@ -72,18 +66,20 @@ The following arguments are supported:
 
 ### pool
 
+~> **Notice** Due to limitations in Terraform, the order of pools in the `linode_lke_cluster` resource is treated as significant.
+For example, the removal of the first listed pool in a cluster may result in all other node pools
+being updated accordingly.
```

Comment:
I'd appreciate some feedback on the wording here, I'm not sure if it gets the point across clearly.

Reply:
I can get the point, but maybe adding a sample diff can help explain the situation better?

Reply:
Maybe we could add a new section explaining these limitations in detail (with examples)?

Reply:
Sounds good 👍